Routes/Question/Qsts.js

```diff
@@ -158,13 +158,27 @@ router.post('/:qstId/Answers', function(req, res) {
       //check if question exists
       if (vld.check(qsts.length, Tags.notFound, null, cb) &&
        vld.hasFields(body, ["guess"], cb)) {
-         if (qsts[0].answer === body.guess) {
-            //add user and question to correct table
-            var prsQst = {personId: req.session.id, questionId: parseInt(qstId)};
-            cnn.chkQry('insert into PersonQuestion set ?', prsQst, cb);
+         // Disregard capitalization
+         if (qsts[0].answer.toLowerCase() === body.guess.toLowerCase()) {
```

Comment:
If you click the "Submit" button on the frontend without providing a guess, it will crash. I get the error: "TypeError: Cannot read property 'toLowerCase' of null". You probably just have to make a check to see if the guess is empty.